File: app/src/main/java/ao/dicionariodetermosinformaticos/romavicdosanjoskc/view/views/DictionaryWordView.java
package ao.dicionariodetermosinformaticos.romavicdosanjoskc.view.views;

import java.util.List;

import ao.dicionariodetermosinformaticos.romavicdosanjoskc.models.DictionaryWordModel;

public interface DictionaryWordView {

    void onSuccessGetListWords(List<DictionaryWordModel> wordModelList);

    void onErrorGetListWords(String messageError);
}

File: app/src/main/java/ao/dicionariodetermosinformaticos/romavicdosanjoskc/view/views/DictionaryWordView.kt
package ao.dicionariodetermosinformaticos.romavicdosanjoskc.view.views

import ao.dicionariodetermosinformaticos.romavicdosanjoskc.models.DictionaryWordModel

interface DictionaryWordView {
    fun onSuccessGetListWords(wordModelList: List<DictionaryWordModel>)
    fun onErrorGetListWords(messageError: String?)
}